card/src/test/java/com/adyen/checkout/card/DefaultCardDelegateTest.kt

```diff
@@ -172,11 +173,31 @@ internal class DefaultCardDelegateTest(
             statesFlow.cancelAndIgnoreRemainingEvents()
         }
 
+        @Test
+        fun `address configuration is full address without default country, then only countries should be emitted`() = runTest {
+            val countriesFlow = addressRepository.countriesFlow.testIn(this)
+            val statesFlow = addressRepository.statesFlow.testIn(this)
+
+            delegate = createCardDelegate(
+                configuration = getDefaultCardConfigurationBuilder()
+                    .setAddressConfiguration(AddressConfiguration.FullAddress())
+                    .build()
+            )
+            delegate.initialize(CoroutineScope(UnconfinedTestDispatcher()))
+
+            assertEquals(TestAddressRepository.COUNTRIES, countriesFlow.awaitItem())
+            assertEquals(Unit, statesFlow.expectNoEvents())
```

Comment:
Why are you trying to test the outcome of `expectNoEvents()`?

Reply:
As when there's no default countries so we're not selecting any country therefore no states request happening here.

Reply:
Yes but this line doesn't make sense, check how we use `expectNoEvents()` in other places